manifests/cron.pp

```diff
@@ -0,0 +1,8 @@
+class centrify::cron {
+  cron { 'flush-centrify-cache':
+    ensure  =>  present,
+    command =>  '/usr/sbin/adflush -f > /dev/null 2>&1',
+    user    =>  root,
+    minute  =>  '*/30',
```

Comment:
I have made the updates

On Fri, Feb 5, 2016 at 4:04 PM, Chadwick Banning <notifications@github.com>
wrote:

> In manifests/cron.pp
> <https://github.com/walkamongus/centrify/pull/3#discussion_r52072378>:
>
> > @@ -0,0 +1,8 @@
> > +class centrify::cron {
> > +  cron { 'flush-centrify-cache':
> > +    ensure  =>  present,
> > +    command =>  '/usr/sbin/adflush -f > /dev/null 2>&1',
> > +    user    =>  root,
> > +    minute  =>  '*/30',
>
> Could the cron times be made configurable? I think if this is added, the
> times should be configurable and the default should be something like once
> a day.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/walkamongus/centrify/pull/3/files#r52072378>.
>